mtp_api/apps/transaction/migrations/0039_consistent_received_at_time.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.10 on 2016-10-25 11:25
+from __future__ import unicode_literals
+from datetime import datetime, time, timedelta
+
+from django.db import migrations
+from django.utils.timezone import utc
+
+
+def make_time_consistent(apps, schema_editor):
+    Transaction = apps.get_model('transaction', 'Transaction')
+    for transaction in Transaction.objects.all():
+        eleven = time(23, 0, 0, tzinfo=utc)
+        midnight = time(0, 0, 0, tzinfo=utc)
+        noon = time(12, 0, 0, tzinfo=utc)
+
+        received_at = transaction.received_at
+        if received_at.timetz() in (eleven, midnight):
+            # check if recorded under BST
+            if received_at.timetz() == eleven:
+                received_at += timedelta(days=1)
+
+            received_at = datetime.combine(received_at.date(), noon)
+            transaction.received_at = received_at
+            transaction.save()
```

Comment:
could it possibly have been done with something like this?

```python
Transaction.objects.filter(received_at__time=time(11, tzinfo=utc)).update(received_at=F(received_at) + timedelta(hours=1))
```

but no big deal as only will run once :)